server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/SensitiveColumnController.java

```diff
@@ -55,6 +56,13 @@ public class SensitiveColumnController {
     @Autowired
     private SensitiveColumnService service;
 
+    @ApiOperation(value = "listColumns", notes = "List table columns and view columns")
+    @RequestMapping(value = "/listColumns", method = RequestMethod.GET)
```

Comment:
`@RequestMapping(value = "/", method = RequestMethod.GET)`
 
return `ListResponse<SensitiveColumn>`

Reply:
`@RequestMapping(value = "/", method = RequestMethod.GET)` should used to list sensitive columns and is occupied yet.